lib/nolibc/stdlib.c

```diff
@@ -431,3 +431,9 @@ void abort(void)
 	uk_pr_crit("Abnormal termination!\n");
 	ukplat_crash();
 }
+
+void exit(int status)
+{
+	uk_pr_info("exit called with status %d, halting system\n", status);
+	ukplat_terminate(status);
+}
```

Comment:
This function is also defined as part of the syscall definition of `exit` in `posix-process`. Either we guard it depending on the presence of `posix-process` or we remove it.